docs/messaging/azure-messaging.md

```diff
@@ -24,12 +24,17 @@ The Semantic Conventions for [Azure Service Bus](https://learn.microsoft.com/azu
 >   * `messaging` - emit the new, stable messaging conventions,
```

Comment:
Should we be crystal clear here and say that this only emits stable conventions? Because it says "stop emitting the old, experimental. I kinda don't like "old" here because there might be "new" unstable conventions as well, and are those included or not? :D 

Maybe we just say: emit only stable messaging conventions. and keep it short. 

Could also be done as a follow up. I just wanted to bring here now for discussions. CC @open-telemetry/specs-semconv-maintainers